app/routes/index.js

```diff
@@ -132,13 +132,40 @@ export default class IndexRoute extends Route {
         sort    : 'starts-at',
         include : 'event-topic,event-sub-topic,event-type,speakers-call',
         filter  : filterOptions
+      }),
+      featuredEvents: await this.store.query('event', {
+        sort    : 'starts-at',
+        include : 'event-topic,event-sub-topic,event-type,speakers-call',
+        filter  : [
+          {
+            and:
+            [
+              {
+                name : 'state',
+                op   : 'eq',
+                val  : 'published'
+              },
+              {
+                name : 'privacy',
+                op   : 'eq',
+                val  : 'public'
+              },
+              {
+                name : 'is-featured',
+                op   : 'eq',
+                val  : true
+              }
+            ]
+          }
+        ]
```

Comment:
This shows all featured events including the ones from past

Reply:
Done

Reply:
Nope. Reuse the returned filterOptions. This will break on search page